here-naksha-lib-handlers/src/main/java/com/here/naksha/lib/handlers/val/EndorsementHandler.java

```diff
@@ -55,14 +58,23 @@ public EndorsementHandler(
     this.properties = JsonSerializable.convert(eventHandler.getProperties(), XyzProperties.class);
   }
 
+  @Override
+  protected EventProcessingStrategy processingStrategyFor(IEvent event) {
+    Request<?> request = event.getRequest();
+    if (request instanceof ContextWriteFeatures<?, ?, ?, ?, ?>) {
+      return PROCESS;
+    }
+    return SUCCEED_WITHOUT_PROCESSING;
```

Comment:
This handler being an intermediate handler, I recommend using SEND_UPSTREAM_WITHOUT_PROCESSING as default behaviour

Reply:
done